examples/todoapp/lib/base/app/config/app_constants.dart

```diff
@@ -13,7 +13,11 @@
 /// https://github.com/FirebaseExtended/flutterfire/blob/4c9b5d28de9eeb5ce76c856fbd0c7b3ec8615e45/docs/messaging/usage.mdx#web-tokens
 library;
 
-
-export 'is_in_test_mode_io_not_available.dart' if (dart.library.io) 'is_in_test_mode_io_available.dart' if (dart.library.html) 'is_in_test_mode_io_not_available.dart';
+export 'is_in_test_mode_io_not_available.dart'
+    if (dart.library.io) 'is_in_test_mode_io_available.dart'
+    if (dart.library.html) 'is_in_test_mode_io_not_available.dart';
 
 const String webVapidKey = '';
+const String translationsKey = 'translations';
```

Comment:
Do we really need to expose feature specific keys to the app_constants? Let's try to keep the app_constants as generic as possible without any feature or data related context

Reply:
Done